pkg/engine/internal/imageverifier.go

```diff
@@ -437,6 +437,10 @@ func (iv *ImageVerifier) buildVerifier(
 	image string,
 	attestation *kyvernov1.Attestation,
 ) (images.ImageVerifier, *images.Options, string) {
+	if attestation.Type == "" && attestation.PredicateType != "" {
+		iv.logger.Info("predicate type has been deprecated, please use type instead")
+		attestation.Type = attestation.PredicateType
+	}
```

Comment:
For now I just added a check that fills Type if `PredicateType` is set but `Type` is not, and replaced all occurrences of `predicateType` with `type`